### PR TITLE
refactor: update dependencies

### PR DESCRIPTION
**Pull request by cshezi**
Updating dependencies
No unit tests are present
- [x] Husky: Eslint fix, Prettier fix

### CODE REVIEW

File: src/start/start.ts [hunk 1]
```diff
@@ -44,16 +45,15 @@ const handleStart = async (options: {
...
       console.log(`Completed Cloning ${repositoryName} for testing`);
 
       console.log(`Started Install dependancy for ${repositoryName}`);
 
-      const intervalId = setInterval(
-        () => updateLoadingIndicator('Installing dependancies...'),
-        150,
-      );
+      const intervalId = setInterval(() => {
+        updateLoadingIndicator('Installing dependancies...');
```
Comment (rtkay123) [inline src/start/start.ts:55]:
```suggestion
        updateLoadingIndicator('Installing dependencies...');
```

File: src/init/template.ts [hunk 2]
```diff
@@ -128,26 +128,33 @@ const handleTemplating = async (): Promise<void> => {
...
   console.log(
     '\nInstalling dependencies has started\nThis might take a minute, Please wait',
   );
-  const intervalId = setInterval(
-    () => updateLoadingIndicator('Installing dependancies...'),
-    150,
-  );
-  shell.cd(response.name);
+  const intervalId = setInterval(() => {
+    updateLoadingIndicator('Installing dependancies...');
```
Comment (rtkay123) [inline src/init/template.ts:155]:
```suggestion
    updateLoadingIndicator('Installing dependencies...');
```

File: src/init/template.ts [hunk 3]
```diff
@@ -197,9 +204,9 @@ const handleTemplating = async (): Promise<void> => {
         `npm i rule@file:${scriptDir}/start/local-rules/frmscoe-${ruleName}-${ruleVersion}.tgz`,
       );
 
-      shell.exec(`npm run build`);
+      shell.exec('npm run build');
 
-      shell.exec(`npm run start`);
+      shell.exec('npm run start');
     } else {
       console.error(`\nError installing dependecy err with ${code}`);
```
Comment (rtkay123) [inline src/init/template.ts:211]:
```suggestion
      console.error(`\nError installing dependency err with ${code}`);
```

**Review by rtkay123 — CHANGES_REQUESTED** [review]
May you delete the [`.eslintignore`](https://github.com/frmscoe/rule-cli/blob/d2ff6e638702d193c4559ddee65a7e302fe64eea/.eslintignore) file, please?

**Review by rtkay123 — APPROVED** [review]

**Review by Sandy-at-Tazama — APPROVED** [review]

**Pull request merged**